src/vs/editor/contrib/inlineCompletions/browser/ghostTextWidget.ts

```diff
@@ -159,16 +160,16 @@ export class GhostTextWidget extends Disposable {
 		this.additionalLinesWidget.updateLines(ghostText.lineNumber, additionalLines, ghostText.additionalReservedLineCount);
 
 		if (notifyUser) {
-			this.audioCueService.playAudioCue(AudioCue.inlineSuggestion).then(() => {
-				if (this.editor.getOption(EditorOption.screenReaderAnnounceInlineSuggestion)) {
-					const lineText = this.editor.getModel()?.getLineContent(ghostText.lineNumber);
-					if (lineText) {
+			const lineText = this.editor.getModel()?.getLineContent(ghostText.lineNumber);
+			if (lineText && this._lastSuggestion !== lineText) {
```

Comment:
I'm wondering whether this will prevent it from reading the same suggestion on a different line, or the same suggestion when triggered separately some time later?

Reply:
Yes, perhaps @hediet can weigh in here. I was trying to fix the issue that reproduces 99% of the time. He may have ideas about what we can do to handle that sort of edge case.